kvmagent/kvmagent/plugins/vms/vm_host_file.py

```
@@ -0,0 +1,209 @@
+            raise ValueError("Tarball must contain exactly one top-level entry, got %d" % len(items))
+
+        if os.path.exists(path):
+            if os.path.isdir(path):
+                shutil.rmtree(path)
+            else:
+                os.remove(path)
+
+        source_content = os.path.join(extract_dir, items[0])
+        shutil.move(source_content, path)
+
+    except Exception as ex:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)